src/api/app/views/layouts/webui/_notification_toolbar.html.haml

```diff
@@ -0,0 +1,11 @@
+- if notification
```

Comment:
I would move the condition [outside](https://github.com/openSUSE/open-build-service/pull/10220/files#diff-45250954406833ba3815d964f479ed8bR48). So if we don't have the notification we don't even call the partial rendering.

Reply:
Moved outside.